integration_tests/utils.py

```diff
@@ -13,6 +13,21 @@ def get_repo_root_path():
     return os.path.dirname(rust_vmm_ci_path)
 
 
+def get_coverage_config_path(platform):
```

Comment:
Since this is not a utility we expect to use in other tests beside the coverage one, we can add this function in the test_coverage.py file.

Reply:
Done.